lib/validators.js

```diff
@@ -1,239 +1,180 @@
 var valid = require('muxrpc-validation')
-var zerr  = require('zerr')
-var ref   = require('ssb-ref')
+var zerr = require('zerr')
+var ref = require('ssb-ref')
 
 // errors
 var MissingAttr = zerr('Usage', 'Param % must have a .% of type "%"')
 var AttrType = zerr('Usage', '.% of param % must be of type "%"')
 
 function isFilter (v) {
-  return (v == '@' || v == '%' || v == '&')
+  return (v === '@' || v === '%' || v === '&')
 }
 
 module.exports = valid({
   msgId: function (v) {
-    if (!ref.isMsg(v))
-      return 'type'
+    if (!ref.isMsg(v)) { return 'type' }
```

Comment:
(a few more of these extra curlies around the place...)

I believe `if (!ref.isMsg(v)) return 'type'` is pretty idiomatic standard code.

Reply:
the ternary example you give would be like writing

```
if(test) {
  then...
} else {
  if(elseIf) {
    then...
  }
}
```
because we don't need to add nesting because we have `else if`.
but ternaries can also do else if!
```
  test ? then
: else_test ? then2
: then3
``` 
as it is, standard understands if ... else if, but not ternary else if. That's a bug in standard, in my opinion.

Reply:
I'll also add: I like to use ternaries where possible because they are an _expression_. an expression returns a value. if I'm using a ternary it implies _this code does not have side effects_
which it turn implies that if regular `if else` is being used, _there may be side effects_.
I think that's a very important layer of meaning.